shared/rakefiles/sync-images/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
```

Comment:
All of the `/sync-images` stuff will move to version-updater repo, right?

Reply:
Yes. I'm waiting to delete it until this PR merges in case people are reading intercommits.

You reminded me that I wanted to prepare that delete in a follow-up PR, so now I have: #355.